### PR TITLE
Fix spelling for release 77.0.

### PR DESCRIPTION
**Pull request by willianrampazzo**
Signed-off-by: Willian Rampazzo <willianr@redhat.com>

### CODE REVIEW

File: avocado/utils/pmem.py [hunk 1]
```diff
@@ -129,7 +129,7 @@ def get_slot_count(self, region):
     def is_region_legacy(region):
         """
         Check whether we have label index namespace. If legacy we can't create
-        new namespaces.
```
Comment (clebergnu) [inline avocado/utils/pmem.py]: I'd argue that it's more natural to keep `namespace` than and add it to the ignore list.

Comment (willianrampazzo) [reply avocado/utils/pmem.py]: I imagined this PR was going to have a comment on this specific term. Looking at it now, namespace makes more sense and is a valid word in the computer science world. I'll change it back and add it to the ignore list.

**Comment by willianrampazzo** [comment]
@clebergnu I force pushed an update with `namespace` added to the ignore list. Let me know if you see something else that needs a change.

File: avocado/utils/pmem.py (hunk 1 above)
Comment (clebergnu) [reply avocado/utils/pmem.py]: Thanks, it LGTM.

BTW, your comment made me think that a "computer science world" dictionary would be a nice thing to be able to use for our checks. Don't know if such a thing exists though.

**Pull request merged**